R/as_gt.R

```diff
@@ -68,7 +68,7 @@ as_gt <- function(
   #     set defaults                              #
   # --------------------------------------------- #
   # set different default title to different methods
-  method <- class(output)[class(output) %in% c("ahr", "wlr", "combo")]
+  method <- class(output)[1]
```

Comment:
I'm a bit worried about the forward-compatibility when using a numeric index to get the class name, since there is no strict rules about the ordering of the class names, the class name vector can easily be `c("somethingelse", "ahr")` unless the object is created in a specific way. So using `%in%` or `match()` may be a more generalizable solution. Does this make any sense?

Reply:
Very nice point! I agree, since `%in%` or `match()` is more generalized. This comment is addressed in the last commit.
(The reason I previously gave up `%in%` is that, when designing the class of a `gs_design_ahr` object, I use `class(x) <- c("ahr", class(x))`. So I feel maybe `class(x)[1]` is better to save the computation.)